static_obstacle_spawner:
  ros__parameters:
    carla_host: "127.0.0.1"
    carla_port: 2000
    publish_rate_hz: 10.0
    inflation_radius: 1.0
    rects:
      r1: { x: -211.7088012695312, y: -95.2505874633789, z: 0.1, yaw: 180.0, length: 4.20, width: 1.60 }
      r2: { x: -273.014831542969, y: -40.430727005004883, z: 0.1, yaw: 90.0,  length: 4.20,  width: 1.60 }
      r3: { x: -273.014831542969, y: 40.430727005004883, z: 0.1, yaw: 90.0,  length: 4.20,  width: 1.60 }
      r4: { x: -225.7088012695312, y: 95.2505874633789, z: 0.1, yaw: 0.0,  length: 4.20,  width: 1.60 }
